apps/camera/js/app.js

```diff
@@ -363,7 +363,9 @@ App.prototype.onBeforeUnload = function() {
  * @private
  */
 App.prototype.loadL10n = function(done) {
-  this.require(['l10n'], done);
+  this.require(['l10n'], function() {
+    this.require(['l10n_date'], done);
+  });
```

Comment:
It is lazily loaded, it isn't in the startup path. I'll check to see if this nested business is plain wrong.